virt-v2v/cold/entrypoint.go

```diff
@@ -62,6 +63,12 @@ func main() {
 		fmt.Println("Error getting XML file:", err)
 		os.Exit(1)
 	}
+	// If needed, customize the VM
+	err = customizeVM(source, xmlFilePath)
```

Comment:
@mnecas @bkhizgiy heads up, `xmlFilePath` will be `yamlFilePath` see:
https://github.com/kubev2v/forklift/pull/778/files#diff-2bd31ec649de5f659c2c9a2f062e677dd9f4bde75f165d281f32ba98c3d9cc68